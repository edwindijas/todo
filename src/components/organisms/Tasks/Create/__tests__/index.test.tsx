import render from '@/utils/test-utils/customRender'
import Component from '..'

describe('Tasks Create', () => {
  it ('matches snapshot', () => {
    const {container} = render(<Component />)
    expect(container).toMatchSnapshot()
  })
})