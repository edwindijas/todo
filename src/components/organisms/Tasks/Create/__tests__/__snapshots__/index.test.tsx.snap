// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tasks Create matches snapshot 1`] = `
<div>
  <div
    class="w-full max-w-lg mx-auto"
  >
    <h1>
      Create Task
    </h1>
    <div
      class="flex flex-col gap-3 py-4"
    >
      <label
        class="undefined  flex relative cursor-text border-[#DCE2ED] bg-[#F9FBFD] fill-blue-desaturated border rounded-lg h-14 px-4"
      >
        <div
          class="relative w-full"
        >
          <span
            class="absolute flex top-0 bottom-0 left-0 items-center px-2 text-blue-desaturated opacity-100"
          >
            Task Name
          </span>
          <input
            class="block outline-none w-full h-full bg-transparent px-2 h-12"
            value=""
          />
        </div>
      </label>
      <label
        class="undefined  flex relative cursor-text border-[#DCE2ED] bg-[#F9FBFD] fill-blue-desaturated border rounded-lg h-14 px-4"
      >
        <div
          class="relative w-full"
        >
          <span
            class="absolute flex top-0 bottom-0 left-0 items-center px-2 text-blue-desaturated opacity-100"
          >
            Description
          </span>
          <input
            class="block outline-none w-full h-full bg-transparent px-2 h-12"
            value=""
          />
        </div>
      </label>
      <label
        class="undefined  flex relative cursor-text border-blue-medium border rounded-lg h-14 px-4"
      >
        <div
          class="relative w-full"
        >
          <span
            class="absolute flex top-0 bottom-0 left-0 items-center px-2 text-blue-medium opacity-100"
          >
            Type
          </span>
          <input
            class="block outline-none w-full h-full bg-transparent px-2 h-12"
            value=""
          />
        </div>
      </label>
      <label
        class="undefined  flex relative cursor-text border-blue-medium border rounded-lg h-14 px-4"
      >
        <div
          class="relative w-full"
        >
          <span
            class="absolute flex top-0 bottom-0 left-0 items-center px-2 text-blue-medium opacity-100"
          >
            Priority
          </span>
          <input
            class="block outline-none w-full h-full bg-transparent px-2 h-12"
            value=""
          />
        </div>
      </label>
      <label
        class="undefined  flex relative cursor-text border-blue-medium border rounded-lg h-14 px-4"
      >
        <div
          class="relative w-full"
        >
          <span
            class="absolute flex top-0 bottom-0 left-0 items-center px-2 text-blue-medium opacity-100"
          >
            Assignee
          </span>
          <input
            class="block outline-none w-full h-full bg-transparent px-2 h-12"
            value=""
          />
        </div>
      </label>
      <label
        class="undefined  flex relative cursor-text border-blue-medium border rounded-lg h-14 px-4"
      >
        <div
          class="relative w-full"
        >
          <span
            class="absolute flex top-0 bottom-0 left-0 items-center px-2 text-blue-medium opacity-100"
          >
            Due Date
          </span>
          <input
            class="block outline-none w-full h-full bg-transparent px-2 h-12"
            value=""
          />
        </div>
      </label>
      <label
        class="undefined  flex relative cursor-text border-blue-medium border rounded-lg h-14 px-4"
      >
        <div
          class="relative w-full"
        >
          <span
            class="absolute flex top-0 bottom-0 left-0 items-center px-2 text-blue-medium opacity-100"
          >
            Attachements
          </span>
          <input
            class="block outline-none w-full h-full bg-transparent px-2 h-12"
            value=""
          />
        </div>
      </label>
      <label
        class="undefined  flex relative cursor-text border-blue-medium border rounded-lg h-14 px-4"
      >
        <div
          class="relative w-full"
        >
          <span
            class="absolute flex top-0 bottom-0 left-0 items-center px-2 text-blue-medium opacity-100"
          >
            labels
          </span>
          <input
            class="block outline-none w-full h-full bg-transparent px-2 h-12"
            value=""
          />
        </div>
      </label>
    </div>
  </div>
</div>
`;
